packages/@n8n/decorators/src/licensed.ts

```diff
@@ -0,0 +1,14 @@
+import { Container } from '@n8n/di';
+
+import { MetadataState } from './metadata-state';
+import type { Controller, BooleanLicenseFeature } from './types';
+
+export const Licensed =
+	(licenseFeature: BooleanLicenseFeature): MethodDecorator =>
```

Comment:
Do we need this for routes in extensions? or could we license-block the entire extension for now, and avoid duplicating `BooleanLicenseFeature` ?

Reply:
Insights will be the first extension and is using `@Licensed` decorators. 

We could have `@n8n/constants` containing `LICENSE_FEATURES` but I'm looking to keep a balance between doing all this refactoring work and making progress on extensions.

Reply:
https://github.com/n8n-io/n8n/pull/14825